variables.tf

```diff
@@ -277,3 +277,34 @@ variable "auto_minor_version_upgrade" {
   default     = null
   description = "Specifies whether minor version engine upgrades will be applied automatically to the underlying Cache Cluster instances during the maintenance window. Only supported if the engine version is 6 or higher."
 }
+
+# Add boolean to create a serverless cluster
```

Comment:
This looks like a leftover comment, used for an AI to generate this variable code :)